### PR TITLE
Fix Compound Rate Issue (#183)

### PR DESCRIPTION
**Pull request by dmelotik**
Fix for https://github.com/messari/subgraphs/issues/183

### CODE REVIEW

**Comment by dmelotik** [comment]
The fix in the compound-v2 subgraph:

image

**Comment by this-username-is-taken** [comment]
great fix!

File: subgraphs/compound-forks/src/mapping.ts [hunk 1]
```diff
@@ -857,9 +857,13 @@ export function snapshotMarket(
   dailySnapshot.exchangeRate = market.exchangeRate;
   dailySnapshot.rewardTokenEmissionsAmount = market.rewardTokenEmissionsAmount;
   dailySnapshot.rewardTokenEmissionsUSD = market.rewardTokenEmissionsUSD;
-  dailySnapshot.rates = market.rates;
   dailySnapshot.blockNumber = blockNumber;
   dailySnapshot.timestamp = blockTimestamp;
+
+  let dailyIdentifier =
```
Comment (0xbe1) [inline subgraphs/compound-forks/src/mapping.ts:863]: i made a helper function for this, look for `getMarketHourlySnapshotID`

**Comment by 0xbe1** [comment]
@dmelotik @this-username-is-taken how about leveraging "deep copy" for reference fields? this way no need to treat interest rate differently

**Pull request merged**